scripts/azureml-assets/azureml/assets/update_assets.py

```diff
@@ -63,6 +65,34 @@ def release_tag_exists(asset_config: assets.AssetConfig, release_directory_root:
     return asset_config.full_name in repo.tags
 
 
+def get_latest_release_tag_version(asset_config: assets.AssetConfig, release_directory_root: Path) -> str:
+    """Check repo to see if an asset's previous version was released if a latest tag exists.
+
+    Args:
+        asset_config (assets.AssetConfig): Asset config
+        release_directory_root (Path): Release branch location
+
+    Returns:
+        str: Latest version found, or None if asset is not an environment or no tags not found
+    """
+    if asset_config.type != AssetType.ENVIRONMENT:
+        return None

```

Comment:
What would you think of moving this check to copy_assets.py? Something like `if check_previous_release and asset_config.type == AssetType.ENVIRONMENT:`

We'll have other asset types that will need similar handling (although not checking via MCR), so keeping this function generic gives us some more options.